oak/server/asylo/asylo_oak_manager.h

```diff
@@ -28,23 +28,20 @@
 
 namespace oak {
 
-class AsyloOakManager final : public Manager::Service {
+class AsyloOakManager {
```

Comment:
Maybe we should call this `AsyloOakLoader` or something like that, and keep the `Manager` name for the gRPC version only, WDYT?